src/main/java/seedu/duke/logic/Logic.java

```diff
@@ -380,11 +382,9 @@ public boolean matchCommand(String command, HashMap<String, String> commandArgum
             break;
         case "delete":
             deleteEntry(commandArguments);
-            storage.update(financialList, budgetLogic);
```

Comment:
Shouldn't this line be left untouched? Since the storage file should be updated when an entry is deleted